EventListener/HttpCacheListener.php

```diff
@@ -114,6 +114,16 @@ public function onKernelResponse(FilterResponseEvent $event)
             $response->setMaxAge($age);
         }
 
+        if (!$response->headers->hasCacheControlDirective('max-stale') && null !== $stale = $configuration->getMaxStale()) {
+            if (!is_numeric($stale)) {
```

Comment:
In which case wouldn't it be a numeric? `setMaxStale` documents `int`, and nothing in the added docs mention the possibility to pass something else.

Reply:
In fact it can be a string with relative time format (1 day, 2 weeks, ...) like for maxage and smaxage.
I updated the phpDoc to clarify this case